pkg/router/template/plugin.go

```diff
@@ -42,6 +42,7 @@ type TemplatePluginConfig struct {
 	StatsUsername          string
 	StatsPassword          string
 	IncludeUDP             bool
+	ValidateRouteTLSConfig bool
```

Comment:
Maybe this should be `ExtendedValidation` which more stringently tests the route against a set of known good safe configuration.